lambda/index.js

```diff
@@ -97,9 +97,12 @@ const ListItemsIntentHandler = {
       const list = items.map((item) => item.name);
 
       // Output
-      speakOutput = `Hier sind deine letzten ${noOfItemsReported}: ${itemsToReport.join(
-        ', '
-      )}`;
+      if(noOfItemsReported == 1) {
```

Comment:
1) For code consistency please insert a space after `if` and before `else`
2) The output in `else` is not always correct

For example: With `maxNoOfItemsReported = 5` and 10 items on the list, the output would be 'Es sind 5 Artikel auf deiner Liste: ...". The original output handled this nicely, I think.

Reply:
2. You are right, I have overlooked that. However I would rather implement another check which checks if you have less than maxNoOfItemsReported on the list. Then it will say the current sentence. If however there are more items on the list, the output could be something like this: "Du hast x Artikel auf deiner Einkaufsliste. Hier sind 5 davon: ..."

Reply:
'Hier sind 5 davon' sounds like you took those randomly. Feel free to update the code, I generally do like the distinction in output.

Reply:
Ok, what about "Die letzten 5 sind: ..." ?
That sounds better.

Reply:
Love it!